### ISSUE TITLE
Bug: CSRF doesn't work properly 

### ISSUE THREAD

**Issue by GreenmeisterDavid** [Barbapapazes/gavarnie#29]
This issue isn't specific for this repo- there are open issues on nuxt-security and nuxt-csurf about this - but I figured I'd still open an issue here as it affects using this template. 

After some time the CRSF-token validation will fail - probably because of serverless setup.

Issue on nuxt-csurf; https://github.com/Morgbn/nuxt-csurf/issues/41

Issue on nuxt-security; https://github.com/Baroshem/nuxt-security/issues/477

**Comment by Barbapapazes** [comment]
Yes, I know this issue and I'm working on a set of modules to create full-stack Nuxt app.

For the moment, I can't do anything appart removing the module.

**Comment by GreenmeisterDavid** [comment]
This might be fixed with this recent nuxt-security update; https://github.com/Baroshem/nuxt-security/pull/532 

Haven't been able to run extensive tests on this yet tho.

**Comment by Barbapapazes** [comment]
Hey 👋,

Yeah, thanks but I have some doubt on the usage.
